app/themes/default/components/ui/breadcrumb.tsx

```diff
@@ -0,0 +1,114 @@
+import { Slot } from '@radix-ui/react-slot'
+import { ChevronRight, MoreHorizontal } from 'lucide-react'
+import * as React from 'react'
+import { cn } from '~/themes/lib/utils'
+
+const Breadcrumb = React.forwardRef<
+  HTMLElement,
+  React.ComponentPropsWithoutRef<'nav'> & {
+    separator?: React.ReactNode
+  }
+>(({ ...props }, ref) => <nav ref={ref} aria-label="breadcrumb" {...props} />)
+Breadcrumb.displayName = 'Breadcrumb'
+
+const BreadcrumbList = React.forwardRef<
+  HTMLOListElement,
+  React.ComponentPropsWithoutRef<'ol'>
+>(({ className, ...props }, ref) => (
+  <ol
+    ref={ref}
+    className={cn(
+      'flex flex-wrap items-center gap-1.5 break-words text-sm text-muted-foreground sm:gap-2.5',
```

Comment:
i think we should rm the break-words , i think it will look better without. standardization is better i guess.

Reply:
hmm... this is an auto generated component by shadcn cli.

I think there's a good reason to keep break-words in certain languages such as German, Tamil and Chinese.
Here's some example from Tailwind https://tailwindcss.com/docs/word-break

Let me know what you think.